quartz_solar_forecast/inverters/enphase/enphase.py

```diff
@@ -1,24 +1,30 @@
 import http.client
 import os
+from typing import Optional
+
 import pandas as pd
 import json
 import base64
 from datetime import datetime, timedelta, timezone
 
-from dotenv import load_dotenv
+from urllib.parse import urlencode
 
-load_dotenv()
+from quartz_solar_forecast.inverters.enphase.enphase_model import EnphaseSettings
```

Comment:
this also leads to the circular import, try to find a workaround where you wont need to import something this way